ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -309,7 +309,9 @@ StackLayout {
                 }
                 onSendMessage: {
                     if (chatInput.fileUrls.length > 0){
-                        chatsModel.sendImage(chatInput.fileUrls[0], false);
+                        chatInput.fileUrls.forEach(url => {
+                            chatsModel.sendImage(url, false);
```

Comment:
What do you think of using `wakuext_sendChatMessages` RPC method to batch send these images?

Reply:
Good idea, implemented.